app/views/people/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<section class="one-view-all">
+  <h2>Person:</h2>
+  <p><%= @person[:name] %></p>
```

Comment:
You can also access model attributes using dot notation, like @person.name, if you want to.